### ISSUE TITLE
RawHtml throws exception

### ISSUE THREAD

**Issue by ivanobulo** [inferinc/bobtail#22]
I was trying to use an example given in documentation:

```
div {}, [rxt.rawHtml('<span>hello</span>')]
```

But received the following exception

> uncaught exception: Cannot insert RawHtml of multiple elements

It is working fine with version 0.0.1 though

**Comment by ivanobulo** [comment]
Looks like it is a CDNJS version issue where the generated code looks like this:

```
if (parsed.length) {
  throw 'Cannot insert RawHtml of multiple elements';
}
```

**Comment by ivanobulo** [comment]
Reopening it. It is in both zip and tar.gz distributions also.

**Comment by yang** [comment]
@ivanobulo Sorry you had to run into that. It's actually already fixed in c6334cf9b6b3bed788a640a46e1661cdea18bb65 (and hence in master). We're putting out a release soon (hopefully this week), but if you don't want to wait, you can either just grab the latest code, or if you use bower you can specify a commit in your bower.json:

``` javascript
"reactive-coffee": "d1275c2c51ccb9716d08489c5aca8b8d0e209342"
```

Let me know if this helps. Thanks for digging into it.

**Comment by ivanobulo** [comment]
I'll do that. thanks for the great project! ;)
